src/main/java/org/broadinstitute/hellbender/utils/codecs/table/TableFeature.java

```diff
@@ -1,26 +1,27 @@
 package org.broadinstitute.hellbender.utils.codecs.table;
 
 
+import htsjdk.samtools.util.Locatable;
 import htsjdk.tribble.Feature;
-import org.broadinstitute.hellbender.utils.GenomeLoc;
-import org.broadinstitute.hellbender.utils.Utils;
 
 import java.util.List;
 
+import static org.broadinstitute.hellbender.utils.Utils.join;
```

Comment:
It creates a worse kind of clutter -- clutter in your namespace. This isn't C, where everyone is used to every file being polluted with random symbols from all over the codebase. Unqualified `join()` in Java code implies (at a first glance) that you've re-implemented a utility method in your own class. It's worth paying the cost of the extra 5 characters `Utils.` to avoid this confusion.